cfgov/v1/tests/util/test_ref.py

```diff
@@ -8,20 +8,6 @@
 )
 
 
-class TestCategories(TestCase):
-    def test_no_duplicate_slugs(self):
```

Comment:
Here's my commit message for this change:

| Now, a page exists (Research Hub Page) that uses the same filterable page categories that are already used by another page (Research Report Page). This is intentional -- the Research Hub page is supposed to filter a subset of Research Reports. So, this test is no longer enforcing desired behavior. Deleting.

If I'm mistaken and something else is going on here, please let me know.

Reply:
It appears I am the one who added this test in https://github.com/cfpb/consumerfinance.gov/pull/2740 ("New unit test that verifies that all page categories have unique keys") but I can't remember a specific reason why this was desirable.